pandas/io/json.py

```diff
@@ -37,6 +41,22 @@ def to_json(path_or_buf, obj, orient=None, date_format='epoch',
     else:
         raise NotImplementedError("'obj' should be a Series or a DataFrame")
 
+    if lines and s[0] == '[' and s[-1] == ']':      # Determine we have a JSON
```

Comment:
more so than the comments on the side? (Determining if we have a JSON list to turn to lines)

I guess I could say that if we are giving a differen json object, then line delimited json doesn't make sense.